app/assets/stylesheets/_flash.css

```diff
@@ -0,0 +1,6 @@
+.flashes{
+   background-color: azure;
+   padding: 5px;
+   border: 1px solid blue;
+   width: fit-content;
+ }
```

Comment:
I think this file was moved. So this file can be removed

Reply:
👌